### PR TITLE
Delayed vesting account original vesting coins modifier

### PR DESCRIPTION
**Pull request by antstalepresh**
## Description

closes: #XXXX

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

File: x/bank/keeper/keeper.go [hunk 1]
```diff
@@ -287,6 +288,29 @@ func (k BaseKeeper) SendCoinsFromAccountToModule(
...
 }
 
+// SendCoinsFromModuleToAccountOriginalVesting does existing bank keeper logic for SendCoinsFromModuleToAccount,
+// except the recipient account should have its original vesting increment if its a vesting account.
+// If recipient is a regular account, just do a normal send
+func (k BaseKeeper) SendCoinsFromModuleToAccountOriginalVesting(
+	ctx sdk.Context, senderModule string, recipientAddr sdk.AccAddress, amt sdk.Coins,
+) error {
+
+	err := k.SendCoinsFromModuleToAccount(ctx, senderModule, recipientAddr, amt)
```
Comment (ValarDragon) [inline x/bank/keeper/keeper.go:298]: We need to check if its a normal account before doing this, right?

Comment (antstalepresh) [reply x/bank/keeper/keeper.go:298]: Vesting account is a kind of normal account isn't it?

Comment (ValarDragon) [reply x/bank/keeper/keeper.go:298]: We confirmed over call, we think this is right. It computes things as bank balance - locked balance, so we have to increment both.

Comment (ValarDragon) [reply x/bank/keeper/keeper.go:298]: We want to add a test for just ensuring these assumptions - nothing becomes immediately spendable, and the full amount is spendable at the end time.

Comment (antstalepresh) [reply x/bank/keeper/keeper.go:298]: Hmm not failing here.
https://github.com/osmosis-labs/cosmos-sdk/pull/4/files#diff-04e4d8ff2ba58020d7cf7f0d80a7505a13719406b6be2ea09ddc402ff412a0e6R829

Comment (antstalepresh) [reply x/bank/keeper/keeper.go:298]: I don't think I understand this test correctly

**Pull request merged**